src/servers/graphql-server.ts

```diff
@@ -184,6 +199,9 @@ export const startApolloServer = async ({
 }): Promise<Record<string, unknown>> => {
   const app = express()
   const httpServer = createServer(app)
+  app.use(cookieParser())
```

Comment:
is that needed? the server itself doesn't need to parse the cookie? (this is done by oathkeeper?)